providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/process/storage/FlagStore.java

```diff
@@ -32,9 +32,16 @@ public class FlagStore implements Storage {
     private final Map<String, FeatureFlag> flags = new HashMap<>();
 
     private final Connector connector;
+    private final boolean throwIfInvalid;
 
     public FlagStore(final Connector connector) {
         this.connector = connector;
+        this.throwIfInvalid = false;
```

Comment:
I would prefer
```suggestion
    public FlagStore(final Connector connector) {
        this(connector, false)
```

so we need to maintain a single constructor